components/Button/src/button.css

```diff
@@ -1,129 +1,144 @@
 .denhaag-button {
-  font-family: var(--denhaag-button-font-family);
+  align-items: center;
+  background-color: var(--denhaag-button-primary-action-background-color);
+  border: 0;
   border-radius: var(--denhaag-button-border-radius);
+  color: var(--denhaag-button-primary-action-color);
+  cursor: pointer;
+  display: inline-flex;
+  font-family: var(--denhaag-button-font-family);
+  font-size: var(--denhaag-typography-scale-base-font-size);
+  line-height: 1.75;
+  padding-block-end: var(--denhaag-button-medium-size-padding-block, var(--denhaag-button-padding-block));
+  padding-block-start: var(--denhaag-button-medium-size-padding-block, var(--denhaag-button-padding-block));
+  padding-inline-end: var(--denhaag-button-medium-size-padding-inline, var(--denhaag-button-padding-inline));
+  padding-inline-start: var(--denhaag-button-medium-size-padding-inline, var(--denhaag-button-padding-inline));
+  position: relative;
 }
 
 .denhaag-button--focus:after,
 .denhaag-button:focus:after {
```

Comment:
I really don't like the CSS2 `:after` notation, think the readability is much better when using the CSS3 `::after` notation.

Reply:
(Same goed for other usages of the `:after` pseudoselector)